data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1701 @@
+			choice
+				`	"So you're fine with letting them study the Pug artifacts with you?"`
+				`	"I can go back and tell them you've agreed then?"`
+			`	"Have you told the others about this?" She asks, and you tell her you haven't. "Alondo, Katya... they need to know about this 'Coalition,' even if they've no intention of invading us. I'll send word to them that you'll be heading to <planet> - you can tell them all the details there, answer their questions. When you're done, you can go pick up these Arachi and bring them here. It'll give me some more time to look at what they've sent me," she says, grabbing some sort of schematic from the suitcase.`
+			`	You say your goodbyes, and you head back to your ship.`
+				accept
+
+mission "House Bebliss 4-FW"
+	name "An Answer to House Bebliss"
+	description "Head back to <destination> to tell House Bebliss that Freya has agreed with their proposal."
+	landing
+	to offer
+		has "House Bebliss 3-FW: done"
+	source "Bourne"
+	destination "Factory of Eblumab"
+	on offer
+		conversation
+			`JJ is waiting for you when you land, wearing an uniform with some medals on his chest. "It was nice to hear about you from Freya. Until she started going on about how you had some 'critical' information, that is," he says as he shakes your hand. "So much for taking a break from managing fleets," he sighs.`
+			choice
+				`	"Sorry about that. It's nice to see you too, JJ. Are the others here yet?"`
+					goto others
+				`	"Well, you won't need to worry about that when it comes to these aliens. Hopefully."`
+			`	"Hopefully?" He gives you a mix between a worried look and a stern face. "Let's just go meet up with the others, that way you don't need to repeat what I'm guessing is a long story."`
+			label others
+			`	He beckons you to follow him to a traffic shuttle. "I brought Katya and Ijs here once we got Freya's message. They were visiting Zug for a presentation about the Conservatory, and that's where I happened to be during my break. Alondo's made some time for us, so I'll bring you to his office. You can tell us the details there."`
+			`	You get in the shuttle, and JJ brings you to a tall government building, with a courtyard in front where various flags fly around the Free Worlds' own. "Planetary flags," JJ says as he gets out of the vehicle. "Some of our planets have been making their own recently, so the diplomacy folks are using that in some buildings." You two head inside the building and go up an elevator, arriving on a floor with a couple rooms to the side and a larger one opposite the elevator. You enter the larger room and see Alondo seated behind a nice wooden desk, Ijs walking around, and Katya sitting on a sofa.`
```

Comment:
I figured that wasn't necessary since JJ is escorting you along the way, and since he's a high ranking FW person you'd just infer he'd have what's necessary to enter the building and head to Alondo's floor.
If you still think it's necessary, I can add a mention somewhere.